nssp/setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import setup
+from setuptools import find_packages
+
+required = [
+    "numpy",
+    "pandas",
+    "pydocstyle",
+    "pytest",
+    "pytest-cov",
+    "pylint==2.8.3",
+    "delphi-utils",
+    "sodapy",
+    "epiweeks",
+    "freezegun",
+    "us",
+]
+
+setup(
+    name="delphi_nssp",
+    version="0.0.1",
```

Comment:
`nssp/delphi_nssp/__init__.py` claims version "0.1.0"...  We should probably make them agree.

Reply:
switching to `0.1.0` to match other initial release indicators. Though I did notice that e.g. `claims_hosp/version.cfg` has `current _version = 0.3.54`, which seems weird given that it's `setup.py` says `0.1.0`

Reply:
oof, thats a problem with our bump2version setup and/or our conventions...  ill make an issue to look at it.

Reply:
https://github.com/cmu-delphi/covidcast-indicators/issues/1962